pom.xml

```diff
@@ -62,6 +62,12 @@
 			<groupId>org.springframework.boot</groupId>
 			<artifactId>spring-boot-starter-test</artifactId>
 		</dependency>
+		<dependency>
```

Comment:
Why do we need this dependency ?

Reply:
This is just a branch from https://github.com/dcsaorg/DCSA-Core/tree/implement-error and merged with master to see how much we already had for this issue.